containerd-shim-v2/stream.go

```diff
@@ -56,7 +56,7 @@ func newTtyIO(ctx context.Context, stdin, stdout, stderr string, console bool) (
 	var err error
 
 	if stdin != "" {
-		in, err = fifo.OpenFifo(ctx, stdin, syscall.O_RDONLY, 0)
+		in, err = fifo.OpenFifo(context.Background(), stdin, syscall.O_RDONLY|syscall.O_NONBLOCK, 0)
```

Comment:
Why create a new context here?

Reply:
The fifo non-blocking goroutine is not in the request scope, so I use a new context.

Reply:
In thus a case, you should use shimv2's service context.

Reply:
Okay.